src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
```

Comment:
The variable name `arr` is not descriptive and decreases code readability. Choose a name that reflects the contents or purpose of the array, such as `styleDeclarations`.